Tribler/Core/Config/config.spec

```diff
@@ -68,6 +68,7 @@ seeding_mode = string(default='ratio')
 seeding_ratio = float(default=2.0)
 seeding_time = float(default=60)
 channel_download = boolean(default=False)
+add_download_to_channel = boolean(default=False)
```

Comment:
I feel like this option should belong to the GUI config, and not to the core config. Instead, the downloading REST API should provide a URL `&` parameter for a download to do it.

It is a cleaner design choice to move user convenience options into the GUI part completely.

Reply:
It is a GUI config but it is also specific to download defaults which I think is a setting that should be in tribler.conf so we can support portable config when we do.